src/main/java/com/xpanxion/java/springboot/da1/demo/controller/student2/BookStoreController.java

```diff
@@ -0,0 +1,32 @@
+package com.xpanxion.java.springboot.da1.demo.controller.student2;
+
+import com.xpanxion.java.springboot.da1.demo.model.student2.Book;
+import com.xpanxion.java.springboot.da1.demo.service.student2.DataAccessStudent2;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class BookStoreController {
+
+    @Autowired
+    private DataAccessStudent2 dataAccess;
+
+    @GetMapping("student2/api/v1/bookstores/1/books")
+    public List<Book> getBookstoreBooks() {
+        int count = 0;
+
+        var bookList = dataAccess.getBookstoreBooks(1);
+        for (Book n : bookList) {
```

Comment:
I'm slightly confused what is going on here.  We need to calculate the total cost of all the books based on the number of books (quantity).  For example, if Barnes and Noble was selling Harry Potter for $3.00 and they had three copies, the total cost would be $9.00 for this book.  Does this PR contain this logic?